src/main/java/uk/gov/companieshouse/accounts/filing/service/accounts/AccountsFilingValidator.java

```diff
@@ -90,11 +92,14 @@ private boolean isValidAccountsType(final String accountsType) {
      *                               entry
      * @param validationStatusErrors - List which holds the validation errors
      */
-    private void validateMadeUpDate(final String madeUpDate, final List<ValidationStatusError> validationStatusErrors) {
+    private void validateMadeUpDate(final String madeUpDate, final List<ValidationStatusError> validationStatusErrors, boolean isOverseas) {
 
         final String madeUpDateMessage = "MadeUpDate : " + madeUpDate;
 
         if (madeUpDate == null || madeUpDate.isBlank()) {
+            if(isOverseas){
+                return;
+            }
```

Comment:
Can we clean up the format here
```suggestion
            if (isOverseas) {
                return;
            }
```

Reply:
suggestion accpeted